pipeline/download_gene.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2021, Blue Brain Project, EPFL
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script that download and postprocess ISH dataset from Allen Brain."""
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+from tqdm import tqdm
+from pathlib import Path
+from typing import Any, Generator, Optional, Tuple
+
+import numpy as np
+import PIL
+from atldld.base import DisplacementField
+
+
+logger = logging.getLogger("download-gene")
```

Comment:
If I am not mistaken we there is not a single log mesage (i.e `logger.info(..)`) in the entire script. IMO it would be useful to add a couple of them